kotlintest-assertions/src/main/kotlin/io/kotlintest/properties/Gen.kt

```diff
@@ -363,6 +380,33 @@ interface Gen<T> {
       override fun random(): Sequence<UUID> = generateSequence { UUID.randomUUID() }
     }
 
+    fun localDate(minYear: Int = 1, maxYear: Int = 2030): Gen<LocalDate> = object : Gen<LocalDate> {
```

Comment:
I think that minYear should default to 1970, but I liked to be able to customize!